### DISCUSSION TITLE
Intermittent issue with app creating tons of new objects on PF11

### DISCUSSION THREAD

**Discussion by Anodarai** [primefaces/community#2397] [PrimeFaces] [answered]
We have a legacy application that's running on RedHat Linux 8, Java 11, Weblogic 14.1.1.1, MyFaces 2.1.18, Primefaces 11.0.
Last year, it was updated from Primefaces 8 to Primefaces 11, and after the update, an intermittent issue appeared where it gets stuck apparently creating and garbage collecting new objects until restart. These are all young generation GC.
In the meantime, we updated from Java 8 to 11, but that did not change anything.
The issue only happens on production environment at our client, we were unable to replicate it on our or theirs test environments. We haven't managed to find what triggers it so far.It looks like our code runs fine, but somewhere in background something is stuck creating new objects.

Maybe someone here encountered a similar issue, or has any tips?

**Comment by melloware** [comment]
Things that have me uneasy about your environment:

1. **MyFaces 2.1.18** is ancient why are you still using it in an EE8 container?
2. **Weblogic 14.1.1.1** is JavaEE8 compliant so you should use MyFaces 2.3.10 which is the version for EE8

**Reply by tandraschko** [reply, marked as answer]
Also try PF14... we fixed and improved so some many stuff in between. nobody will help you to improve a application with PF11.
